types/jest/index.d.ts

```diff
@@ -939,26 +940,22 @@ declare namespace jest {
          * });
          *
          */
-        mockRejectedValueOnce(value: any): Mock<T>;
+        mockRejectedValueOnce(value: any): Mock<Promise<T>, Y>;
     }
 
     /**
      * Represents the result of a single call to a mock function.
      */
     interface MockResult {
+        type: 'return' | 'throw' | 'incomplete';
         /**
-         * True if the function threw.
-         * False if the function returned.
-         */
-        isThrow: boolean;
```

Comment:
Context for this change? Was this property removed from Jest?

I see isThrow still in the docs, and no 'type' at all: https://jestjs.io/docs/en/mock-function-api#mockfnmockresults